modules/ossm-exposing-service-using-istio-gateway-and-virtualservice.adoc

```diff
@@ -0,0 +1,156 @@
+// This procedure is used in the following assembly:
+// * gateways/ossm-getting-traffic-into-a-mesh.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-exposing-service-using-istio-gateway-and-virtualservice_{context}"]
+= Exposing a service using the {istio} gateway and VirtualService resources
+:context: ossm-exposing-service-using-istio-gateway-and-virtualservice
+
+This procedure uses the {istio} `Gateway` and `VirtualService` resources to configure a gateway that was deployed using gateway injection. The resources configure the gateway to expose a service in the mesh to traffic outside the mesh. Then, the gateway is exposed to traffic outside the cluster by setting the `Service` for the gateway to type `LoadBalancer`.
+
+.Prerequisites
+
+* You have installed an {istio} gateway using gateway injection.
+
+.Procedure
+
+. Deploy a sample service in a mesh namespace. 
```

Comment:
May be we can rephrase this sentence to

`Deploy a sample service, named httpbin, in a mesh namespace.`

OR 

`Deploy an httpbin service in a mesh namespace.`

Reply:
done